packages/core/core/src/types.ts

```diff
@@ -378,7 +380,7 @@ REVIEWS_SEARCH_PARAMS,
 REVIEW_ADD_PARAMS,
 API extends PlatformApi = any
 > extends Composable<API> {
-  search(params: ComposableFunctionArgs<REVIEWS_SEARCH_PARAMS>): Promise<void>;
+  search(params: ComposableFunctionArgs<REVIEWS_SEARCH_PARAMS>, force?: boolean): Promise<void>;
```

Comment:
Again some of them has cacheTimestamp and some does not. Why?

Reply:
Added in [d65b0a7](https://github.com/vuestorefront/vue-storefront/pull/6420/commits/d65b0a7f532d7c4e40cdb945f7a43a82c568b2e7). By the way, some of them had and some didn't because for example `useProduct` and `useReview` interfaces contained an index signature `[x: string]: any;` and were more flexible in that regard.